kratos/python/add_matrix_to_python.cpp

```diff
@@ -76,6 +76,22 @@ namespace Python
         //here we add the dense matrix
         auto matrix_binder = CreateMatrixInterface< DenseMatrix<double> >(m,"Matrix");
         matrix_binder.def(py::init<const DenseMatrix<double>::size_type, const DenseMatrix<double>::size_type>());
+        matrix_binder.def("__init__", [](DenseMatrix<double> &m, py::buffer b){
+          py::buffer_info info = b.request();
+          KRATOS_ERROR_IF( info.format != py::format_descriptor<double>::value ) << "Expected a double array\n";
```

Comment:
```suggestion
          KRATOS_ERROR_IF(info.format != py::format_descriptor<double>::value) << "Expected a double array" << std::endl;
```

Reply:
So the `\n` is not encouraged? I will change it

Reply:
do you know the difference btw `"\n"` and `std::endl`? => afaik the only difference is that `std::endl` flushes the output (https://stackoverflow.com/a/213977)
Since you throw you want to get the output, hence flushing makes sense

Plus I think we do this in most cases, hence I would stick to it

Reply:
yes and because it is flushed anyway after the error is thrown, I wanted to save the seven extra characters. But I do not depend on it, I changed it